src/App.js

```diff
@@ -1,10 +1,9 @@
 import React, { useState } from 'react';
 import './App.css';
-
 import Board from './components/Board';
 
-const PLAYER_1 = 'X';
-const PLAYER_2 = 'O';
+const PLAYER_1 = 'X'
+const PLAYER_2 = 'O'
```

Comment:
By changing your characters for players to lowercase your tests will pass. I know, annoying.